spec/models/condition_spec.rb

```diff
@@ -166,11 +166,11 @@
       end
 
       it 'should work' do
-        expect(@cond.to_s).to eq "Question #1 is less than 5"
+        expect(@cond.to_s).to eq "Question #0 is less than 5"
```

Comment:
Zoiks, this is one place where we need sensible ranks. These strings are shown to the user.